com.unity.netcode.gameobjects/package.json

```diff
@@ -6,6 +6,6 @@
     "unity": "2020.3",
     "dependencies": {
         "com.unity.nuget.mono-cecil": "1.10.1",
-        "com.unity.collections": "1.1.0"
+        "com.unity.transport": "1.0.0-pre.15"
```

Comment:
I like the fact that we won't be depending on collections, jobs, burst on our own directly but depend on UTP instead and keep things consistent between two <3

Reply:
I was thinking that we don't have a requirement on a specific collections package version and if we were to have a specific version dependency, we could add it here later, up until then, we could just rely on UTP's.
However, that means UTP would indirectly break NGO by upgrading collections package version on their end and we'd retroactively specify a specific version of collections package after the fact.
So, I get your point and we'll keep collections 1.1.0 in the package.json — very good catch!